packages/flutter/test/foundation/service_extensions_test.dart

```diff
@@ -373,37 +406,66 @@ void main() {
   });
 
   test('Service extensions - platformOverride', () async {
+    final Iterable<Map<String, dynamic>> extensionChangedEvents =
```

Comment:
this would be a case where exceeding >80 chars is reasonable. I would also avoid the extra line break between this line and the following line. the two lines are actually somewhat related so no need for extra whitespace./

Reply:
done.